providers/Flipt/README.md

```diff
@@ -0,0 +1,60 @@
+# FliptProvider for OpenFeature PHP SDK
+
+The `FliptProvider` is an integration for the [OpenFeature](https://github.com/open-feature/php-sdk) PHP SDK, allowing you to use [Flipt](https://flipt.io) as a feature flagging service.
+
+## Installation
+
+Before using the `FliptProvider`, you must have the OpenFeature PHP SDK installed. If you have not already installed the SDK, you can do so with Composer:
+
+```bash
+composer require open-feature/php-sdk
+```
+
+Next, include the FliptProvider in your project:
+
+```bash
+composer require open-feature/flipt-provider
+```
+
+## Usage
+To use the FliptProvider, you'll need to create an instance of it by passing your Flipt host, API token, and namespace (if applicable). Then, set this provider for OpenFeature.
+
+Here's a quick example:
+
+```php
+
+use OpenFeature\OpenFeature;
+use OpenFeature\Providers\Flipt\FliptProvider;
+
+// Replace these with your actual Flipt host and API token
+$host = 'http://your-flipt-instance';
+$apiToken = 'your-api-token';
+$namespace = 'namespace';
+
+$provider = new FliptProvider($host, $apiToken, $namespace);
+
+OpenFeature::setProvider($provider);
+
+// Now you can evaluate your feature flags as follows
+$booleanFlagValue = OpenFeature::getBooleanValue('your-boolean-flag-key', false);
+$stringFlagValue = OpenFeature::getStringValue('your-string-flag-key', 'default-value');
+$integerFlagValue = OpenFeature::getIntegerValue('your-integer-flag-key', 0);
+$floatFlagValue = OpenFeature::getFloatValue('your-float-flag-key', 0.0);
+$objectFlagValue = OpenFeature::getObjectValue('your-object-flag-key', ['default' => 'value']);
```

Comment:
You need to retrieve a client currently, these resolve methods are not on the global API.

Reply:
Instead of `OpenFeature::getBoolean` you would do `$client = OpenFeature::getInstance()->get client()` and interact with that.

Reply:
A facade global might be a useful functionality but it's not an available one right now 🙂

Reply:
Actually I copied the wrong code. Thats the usage when using my [laravel openfeature package](https://github.com/legoheld/openfeature-laravel) ;)

There I registered the OpenFeature client directly under a facade for easy access.